hermes-api/src/main/java/pl/allegro/tech/hermes/api/TrackingMode.java

```diff
@@ -0,0 +1,8 @@
+package pl.allegro.tech.hermes.api;
+
+public enum TrackingMode {
```

Comment:
One more thing, it would be great not to use this enum values in API. instead we could use some strings like "trackingAll", "discardedOnly", "trackingOff" and construct proper enum values from them. this way we'll be able to change the internal any time without crashing APIs.